File: app/src/main/java/id/poncoe/latihanmasterandroidponcoe/java/storage/mypreloaddata/MahasiswaActivity.java
package id.poncoe.latihanmasterandroidponcoe.java.storage.mypreloaddata;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import java.util.ArrayList;

import id.poncoe.latihanmasterandroidponcoe.R;
import id.poncoe.latihanmasterandroidponcoe.java.storage.mypreloaddata.adapter.MahasiswaAdapter;
import id.poncoe.latihanmasterandroidponcoe.java.storage.mypreloaddata.database.MahasiswaHelper;
import id.poncoe.latihanmasterandroidponcoe.java.storage.mypreloaddata.model.MahasiswaModel;

public class MahasiswaActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_mahasiswa);

        RecyclerView recyclerView = findViewById(R.id.recyclerview);
        recyclerView.setLayoutManager(new LinearLayoutManager(this));
        MahasiswaAdapter mahasiswaAdapter = new MahasiswaAdapter();
        recyclerView.setAdapter(mahasiswaAdapter);

        MahasiswaHelper mahasiswaHelper = new MahasiswaHelper(this);
        mahasiswaHelper.open();
        // Ambil semua data mahasiswa di database
        ArrayList<MahasiswaModel> mahasiswaModels = mahasiswaHelper.getAllData();
        mahasiswaHelper.close();

        mahasiswaAdapter.setData(mahasiswaModels);
    }
}

File: app/src/main/java/id/poncoe/latihanmasterandroidponcoe/kotlin/storage/mypreloaddata/MahasiswaActivity.kt
package id.poncoe.latihanmasterandroidponcoe.kotlin.storage.mypreloaddata

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.recyclerview.widget.LinearLayoutManager
import id.poncoe.latihanmasterandroidponcoe.kotlin.storage.mypreloaddata.adapter.MahasiswaAdapter
import com.dicoding.picodiploma.mypreloaddata.database.MahasiswaHelper
import id.poncoe.latihanmasterandroidponcoe.R
import kotlinx.android.synthetic.main.activity_mahasiswa.*

class MahasiswaActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_mahasiswa)

        recyclerview.layoutManager = LinearLayoutManager(this)
        val mahasiswaAdapter = MahasiswaAdapter()
        recyclerview.adapter = mahasiswaAdapter

        val mahasiswaHelper = MahasiswaHelper(this)
        mahasiswaHelper.open()
        // Ambil semua data mahasiswa di database
        val mahasiswaModels = mahasiswaHelper.getAllData()
        mahasiswaHelper.close()

        mahasiswaAdapter.setData(mahasiswaModels)
    }
}
